cli/deno_error.rs

```diff
@@ -46,10 +46,14 @@ impl fmt::Display for StaticError {
   }
 }
 
-pub fn bad_resource() -> ErrBox {
+pub fn bad_resource(_rid: u32) -> ErrBox {
```

Comment:
It'd be cool if we could annotate the error message with actual `rid` but for now I'm getting errors:
```
error[E0716]: temporary value dropped while borrowed
  --> cli/deno_error.rs:54:40
   |
54 |   StaticError(ErrorKind::BadResource, &format!("bad resource kind {:?}", rid.clone())).into()
   |                                       -^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |                                       ||
   |                                       |creates a temporary which is freed while still in use
   |                                       requires that borrow lasts for `'static`
55 | }
   | - temporary value is freed at the end of this statement
```

This can be figured out in separate PR tho

Reply:
It's probably because `StaticError` requires a `&'static str`. I think there are other ways to construct an ErrBox which don't require a `&'static str` error message. But I agree that this can be done later.